cli/command/load_balancer.go

```diff
@@ -223,12 +232,17 @@ func (l *LoadBalancerCommand) create(c *cli.Context) error {
 
 	req := models.CreateLoadBalancerRequest{
 		LoadBalancerName: args["LOAD_BALANCER_NAME"],
+		LoadBalancerType: models.LoadBalancerType(c.String("type")),
 		EnvironmentID:    environmentID,
 		IsPublic:         !c.Bool("private"),
 		Ports:            ports,
 		HealthCheck:      healthCheck,
 	}
 
+	if err := req.Validate(); err != nil {
```

Comment:
don't do that here; this already happens in the `client` package.

Reply:
✅